scalabel/vis/label.py

```diff
@@ -490,6 +490,18 @@ def parse_args() -> argparse.Namespace:
         "output folder instead of being displayed "
         "interactively.",
     )
+    parser.add_argument(
+        "--range_begin",
```

Comment:
```suggestion
        "--range-begin",
```
Most arguments use dash instead of underscores in command-line options. Please fix the inconsistencies in other `add_argument` as well.